tracer/src/Datadog.Trace/ISpan.cs

```diff
@@ -3,6 +3,8 @@
 // This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
 // </copyright>
 
+#nullable enable
```

Comment:
I wonder if anything else here should actually be nullable, e.g. `OperationName`. I'm not sure if it would be flagged by the build, because we're typically not interacting with `ISpan`, just `Span`

Reply:
Done in https://github.com/DataDog/dd-trace-dotnet/pull/5656 (along with discussion)